com.chartboost.mediation/Runtime/AdFormats/Banner/ChartboostMediationBannerViewAndroid.cs

```diff
@@ -0,0 +1,160 @@
+#if UNITY_ANDROID
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Chartboost.Banner;
+using Chartboost.Events;
+using Chartboost.Requests;
+using Chartboost.Results;
+using Chartboost.Utilities;
+using Newtonsoft.Json;
+using UnityEngine;
+using Logger = Chartboost.Utilities.Logger;
+
+namespace Chartboost.AdFormats.Banner
+{
+    internal class ChartboostMediationBannerViewAndroid : ChartboostMediationBannerViewBase
+    {
+        private readonly AndroidJavaObject _bannerAd;
+        internal Later<ChartboostMediationBannerAdLoadResult> LoadRequest;
+
+        private Dictionary<string, string> _keywords = new Dictionary<string, string>();
+
+        public ChartboostMediationBannerViewAndroid(AndroidJavaObject bannerAd) : base(new IntPtr(bannerAd.HashCode()))
+        {
+            LogTag = "ChartboostMediationBanner (Android)";
+            _bannerAd = bannerAd;
+            
+            // TODO: this can be also done on bridge?   
+            AndroidAdStore.TrackBannerAd(bannerAd);
+        }
+
+        public override Dictionary<string, string> Keywords
+        {
+            get => _keywords;
+            set
+            {
+                try
+                {
+                    _bannerAd.Call("setKeywords", value.ToKeywords());
+                    _keywords = value;
+                }
+                catch (Exception e)
+                {
+                    EventProcessor.ReportUnexpectedSystemError($"Error setting keywords => {e.Message}");
+                }
+            }
+        }
+
+        public override ChartboostMediationBannerAdLoadRequest Request { get; protected set; }
+
+        public override BidInfo WinningBidInfo
+        {
+            get => _bannerAd.Get<AndroidJavaObject>("winningBidInfo").MapToWinningBidInfo();
+            protected set { }
+        }
+
+        public override string LoadId
+        {
+            get => _bannerAd.Get<string>("loadId");
+            protected set { }
+        }
+
+        // Note: This is currently only available in iOS but not on Android   
+        // Android will include this from 5.0, public API `IChartboostMediationBannerView` will then include this field as well
+        public override Metrics? LoadMetrics
+        {
+            get => null;
```

Comment:
Wait Metrics aren't available on Android for Adaptive Banners? 🤔 but they are for iOS. That does not sound correct. @kushG

Reply:
Android is going to update their Banner API(similar to Fullscreen API) in 5.0 so they will have it available then. @CB-RyanMcCormick 

Also, currently, it is not available publicly but only internally

Reply:
iOS brought their banner API inline with their Fullscreen API for this version. Their old implementation allowed them to layer this change over the old one. Ours would only be doable with a breaking change, so it will wait for 5.0